packages/fxa-settings/src/components/FormVerifyCode/index.tsx

```diff
@@ -81,14 +82,16 @@ const FormVerifyCode = ({
     }
   };
 
-  const { handleSubmit, register, errors } = useForm<FormData>({
+  const { handleSubmit, register, errors, watch } = useForm<FormData>({
     mode: 'onBlur',
     criteriaMode: 'all',
     defaultValues: {
       code: '',
     },
   });
 
+  const codeValue = watch('code');
```

Comment:
This is really the main change, everything else is getting tests working for the new functionality.